packages/aws-rfdk/lib/deadline/README.md

```diff
@@ -236,14 +284,15 @@ _**Note:** This construct is not usable in any China region._
 The following example outlines how to construct `UsageBasedLicensing`:
 
 ```ts
-const recipes = new ThinkboxDockerRecipes(stack, 'Recipes', {
-  stage: Stage.fromDirectory(/* ... */)
+const version = VersionQuery.exactString(stack, 'Version', '1.2.3.4');
```

Comment:
AFAIK, we don't have `exactString()` anymore.

Reply:
Changed this to use the `VersionQuery` constructor

Reply:
Confirmed this occurrence is fixed. We will have to come back and address the other occurrences later. Resolving.